### ISSUE TITLE
Support popular benchmarks

### ISSUE THREAD

**Issue by penguine-ip** [confident-ai/deepeval#508]
Currently, users of `deepeval` can only create their own evaluation dataset/test cases. The support more users fine-tuning their model, `deepeval` should be able to import standard benchmarks such as MMLU, hellaswag, TruthfulQA, Big Bench, etc.

Comment or DM on discord to discuss more: https://discord.com/invite/a3K9c8GRGt

**Comment by Peilun-Li** [comment]
+1. A paved path for generating metrics/scores covered by [Open LLM Leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard) and [LLM Safety Leaderboard](https://huggingface.co/spaces/AI-Secure/llm-trustworthy-leaderboard) (and maybe a few others) can be a great bonus for the framework

**Comment by penguine-ip** [comment]
@rohinish404 We're trying to support multiple popular benchmarks for users to have instant access to through deepeval. TruthfulQA is a popular and also extremely easy to integrate: https://github.com/sylinrl/TruthfulQA/blob/main/data/v0/TruthfulQA.csv

If users can just have access to truthfulQA like this:
```python
from deepeval.benchmarks import TruthfulQA

truthful_qa = TruthfulQA()
truthful_qa.evaluate(...)
```

In .benchmarks:
```python
from deepeval.dataset import EvaluationDataset

class TruthfulQA(EvaluationDataset):
    pass
```

Does this make sense to you?

**Comment by rohinish404** [comment]
> @rohinish404 We're trying to support multiple popular benchmarks for users to have instant access to through deepeval. TruthfulQA is a popular and also extremely easy to integrate: https://github.com/sylinrl/TruthfulQA/blob/main/data/v0/TruthfulQA.csv
> 
> If users can just have access to truthfulQA like this:
> 
> ```python
> from deepeval.benchmarks import TruthfulQA
> 
> truthful_qa = TruthfulQA()
> truthful_qa.evaluate(...)
> ```
> 
> In .benchmarks:
> 
> ```python
> from deepeval.dataset import EvaluationDataset
> 
> class TruthfulQA(EvaluationDataset):
>     pass
> ```
> 
> Does this make sense to you?

Yup let me have a look at the codebase and then will ask further in discord.

**Comment by JiwaniZakir** [comment]
I'd like to work on this issue. I'll submit a PR shortly.

**Comment by JiwaniZakir** [comment]
I've submitted a PR to address this issue: https://github.com/confident-ai/deepeval/pull/2527

**Comment by penguine-ip** [comment]
Hey @JiwaniZakir i believe users can already import from `deepeval.benchmarks`?

**Comment by JiwaniZakir** [comment]
This caught my eye. I think I see what's going on here, let me put together a fix.

**Comment by JiwaniZakir** [comment]
Picked this up -- working on a fix now.

**Comment by JiwaniZakir** [comment]
PR is up: https://github.com/confident-ai/deepeval/pull/2539

**Comment by HarshaVardhanRepudi** [comment]
Hi @penguine-ip — I'd like to add MMLU-Pro to complete the Open LLM Leaderboard v2 coverage. I'll model it on the existing mmlu/ module. Does that fit what you had in mind?

**Comment by HarshaVardhanRepudi** [comment]
Opened #2980 adding MMLU-Pro for this.
